Procfile

```diff
@@ -1,2 +1,3 @@
 web: bundle exec puma -C config/puma.rb
+worker: bundle exec sidekiq -c 8
```

Comment:
@littleforest do we also want this in `Procfile.dev` or just for prod?

Reply:
Good question! Does Rails just use inline for dev environment by default? If so, should we stick with that for the short-term so devs don't need to get Redis installed?

Reply:
Sounds good!